### PR TITLE
General code and packaging cleanup 

### PR DESCRIPTION
**Pull request by rockfordlhotka**
* Fix XML comment warnings
* Move output directory of nuget packages
* Add MAUI to nuget output
* Attempt to add Blazor to nuget output

@luizfbicalho in #4058 you got the nuget packages to build, but for some reason the two Blazor projects aren't actually generating any output. Any idea why this is happening?

### CODE REVIEW

**Comment by luizfbicalho** [comment]
I'll take a look

**Comment by luizfbicalho** [comment]
I created this PR #4118 with a solution , see if now works for you

File: Source/Csla.Blazor.WebAssembly/Csla.Blazor.WebAssembly.csproj
Comment (luizfbicalho) [inline Source/Csla.Blazor.WebAssembly/Csla.Blazor.WebAssembly.csproj:1]: I removed those lines

**Comment by rockfordlhotka** [comment]
> I created this PR #4118 with a solution , see if now works for you

That seems to have fixed it, thank you!

File: Source/Csla.Data.SqlClient/Csla.Data.SqlClient.csproj [hunk 1]
```diff
@@ -26,6 +26,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.12.0" />
```
Comment (StefanOssendorf) [inline Source/Csla.Data.SqlClient/Csla.Data.SqlClient.csproj:29]: Why is this necessary?

File: Source/Csla.Xaml.Maui/Csla.Xaml.Maui.csproj [hunk 2]
```diff
@@ -1,6 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\Directory.Package.props" />
```
Comment (StefanOssendorf) [inline Source/Csla.Xaml.Maui/Csla.Xaml.Maui.csproj:2]: Is that necessary? Something MAUI specific?

File: Source/Csla.Analyzers/Csla.Analyzers/AsynchronousBusinessRuleInheritingFromBusinessRuleAnalyzer.cs [hunk 3]
```diff
@@ -7,6 +7,9 @@
 
 namespace Csla.Analyzers
 {
+  /// <summary>
```
Comment (StefanOssendorf) [inline Source/Csla.Analyzers/Csla.Analyzers/AsynchronousBusinessRuleInheritingFromBusinessRuleAnalyzer.cs:10]: Why aren't you just disabling the doc generation for this particular project? Empty comments have no value at all?

File: Source/Csla.Data.SqlClient/Csla.Data.SqlClient.csproj (hunk 1 above)
Comment (rockfordlhotka) [reply Source/Csla.Data.SqlClient/Csla.Data.SqlClient.csproj:29]: This was necessary to resolve a package vulnerability warning - I forced it to use the non-vulnerable version.

File: Source/Csla.Xaml.Maui/Csla.Xaml.Maui.csproj (hunk 2 above)
Comment (rockfordlhotka) [reply Source/Csla.Xaml.Maui/Csla.Xaml.Maui.csproj:2]: The project type? That is MAUI specific.

File: Source/Csla.Analyzers/Csla.Analyzers/AsynchronousBusinessRuleInheritingFromBusinessRuleAnalyzer.cs (hunk 3 above)
Comment (rockfordlhotka) [reply Source/Csla.Analyzers/Csla.Analyzers/AsynchronousBusinessRuleInheritingFromBusinessRuleAnalyzer.cs:10]: Would that have disabled the compiler warnings?
